app/src/main/java/com/edulectronics/tinycircuit/Controllers/WireController.java

```diff
@@ -79,4 +84,36 @@ private ConnectionPoint getConnectionPoint(Component component, ConnectionPointO
         }
         return null;
     }
+
+    public List<Wire> getWires(Point a, Point b) {
+        List<Wire> wires = new ArrayList<>();
+        Point lastPoint = a;
+
+        for (int i = 0; i <= 2; i++) {
+            Wire w;
+            if (i % 2 == 0) {
+                w = getVerticalWire(lastPoint, b);
+            } else {
+                w = getHorizontalWire(lastPoint, b);
+            }
+
+            if (w != null) {
+                lastPoint = w.b;
+                wires.add(w);
+            }
+        }
```

Comment:
Nu wordt altijd eerst een verticale en dan een horizontale wire opgehaald. Het lijkt m dat dit afhankelijk moet worden van de orientatie van het connectionpoint (bijv connectionpoint links, dan eerst de horizontale wire)
Daarnaast afhankelijk van de oriantaties van connectionpoints ten opzichte van elkaar. Stel: twee componenten naast elkaar, links en rechts. De Linker connectionpoint van het linker component is verbonden met het rechter component van het rechter connectionpoint. Nu moet de wire vanaf links eerst omhoog/omlaag, en dan links, om om het linker component heen te gaan.
Echter als de componenten hun andere connectionpoints hadden verbonden kun je wel meteen horizontaal gaan.